test/swingsetTests/gallery/vat-alice.js

```diff
@@ -369,6 +368,38 @@ function makeAliceMaker(E, log, contractHost) {
           showPaymentBalance('childPurseP', childPurseP);
           showPaymentBalance('grandchildPurseP', grandchildPurseP);
         },
+        async doGetAllPixels() {
+          log('++ alice.doGetAllPixels starting');
+          // No errors with this
+          // const pixels = [];
```

Comment:
Does the comment ('No errors with this') reflect the current state or a goal? Is the commented out code intended to be reinstated? If so, this deserves a TODO. 

I can't tell whether the code used to pass incorrectly, would currently pass, but shouldn't , or is expected to pass in some future state. I guess it needs a more explicit description.

Reply:
Good catch, I'll take that out